Recipes/Recipes/ProfileModule/View/ProfileView.swift

```diff
@@ -0,0 +1,196 @@
+        static let cancelAlert = "Cancel"
+        static let okAlert = "Ok"
+        static let titleAlert = "Change your name and surname"
+        static let placeholderText = "Name Surname"
+    }
+    
+    enum TypeCells {
+        case profileAvatar
+        case profileBonuses
+        case profilePolicy
+        case profileLogOut
+    }
+
+    // MARK: - Visual Components
+    
+    private let tableView: UITableView = {
+       let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+
+    // MARK: - Public Properties
+
+    var presenter: ProfilePresenter?
+
+    // MARK: - Private Properties
+    
+    private let typeCells: [TypeCells] = [.profileAvatar, .profileBonuses, .profilePolicy, .profileLogOut]
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        configureUI()
+    }
+
+    // MARK: - Private Methods
+    
+    private func setupTableView() {
+        view.addSubview(tableView)
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.separatorStyle = .none
+        tableView.isScrollEnabled = false
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(
+            ProfileAvatarTableViewCell.self, 
+            forCellReuseIdentifier: 
+                ProfileAvatarTableViewCell.Constants.identifier
+        )
+        tableView.register(
+            ProfileBonusesTableViewCell.self,
+            forCellReuseIdentifier:
+                ProfileBonusesTableViewCell.Constants.identifier
+        )
+        tableView.register(
+            ProfilePolicyTableViewCell.self,
+            forCellReuseIdentifier:
+                ProfilePolicyTableViewCell.Constants.identifier
+        )
+        tableView.register(
+            LogOutTableViewCell.self,
+            forCellReuseIdentifier:
+                LogOutTableViewCell.Constants.identifier
+        )
+        
+    }
+
+    private func configureUI() {
+        view.backgroundColor = .white
+        tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor).isActive = true
+        tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
+        tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+    }
+    
+    private func logOutAlert() {
+        let alert = UIAlertController(title: Constants.alertTitle, message: nil, preferredStyle: .alert)
+        let okAction = UIAlertAction(title: Constants.yesAlert, style: .default)
+        let actionCancel = UIAlertAction(title: Constants.cancelAlert, style: .default)
+        alert.addAction(okAction)
+        alert.addAction(actionCancel)
+        present(alert, animated: true)
```

Comment:
```suggestion
        present(alert, animated: true)
```
alert в экстеншен как у Сонаты